### PR TITLE
Update Default names of Triangle and Vertex Arrays in Triangle Geometry and fix plugin unit test

### PR DESCRIPTION
**Pull request by imikejackson**
This helps the data structure be a bit more familiar to those coming from SIMPL.

Minor filter parameters organization for ExtractInternalSrufacesFromTriangleGeometry
and LinkGeometryDataFilter.

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

### CODE REVIEW

File: cmake/Plugin.cmake [hunk 1]
```diff
@@ -83,10 +83,12 @@ function(complex_add_plugin)
     complex_COMPILE_PLUGIN(PLUGIN_NAME ${ARGS_PLUGIN_NAME}
                            PLUGIN_SOURCE_DIR ${COMPLEX_${ARGS_PLUGIN_NAME}_SOURCE_DIR}
     )
-    # Increment the plugin count
-    get_property(PLUGIN_COUNT GLOBAL PROPERTY COMPLEX_PLUGIN_COUNT)
-    math(EXPR PLUGIN_COUNT "${PLUGIN_COUNT}+1")
-    set_property(GLOBAL PROPERTY COMPLEX_PLUGIN_COUNT ${PLUGIN_COUNT})
+    if(COMPLEX_PLUGIN_ENABLE_${ARGS_PLUGIN_NAME})
```
Comment (JDuffeyBQ) [inline cmake/Plugin.cmake:86]: This seems unrelated.

Comment (imikejackson) [reply cmake/Plugin.cmake:86]: it isn't related to the PR or the code isn't related to the code above?

Comment (JDuffeyBQ) [reply cmake/Plugin.cmake:86]: The PR.
